src/ProfiledAot/src/CommonMethods.cs

```diff
@@ -19,6 +21,10 @@ public static async Task<string> Invoke()
         foo.StartsWith("f");
         foo.Contains("o");
 
+        string someString = MauiTest.Resources.Strings.SomeString;
+
+        InitializeCommonEssentials();
+
         using var client = new HttpClient();
         var send = client.SendAsync (new HttpRequestMessage (HttpMethod.Get, url));
         var getstring = client.GetStringAsync (url);
```

Comment:
This looks good, but maybe should add: `Color.TryParse("White")` and `Color.TryParse("#ffffff")`

Reply:
Good idea. Done.